crates/test-programs/wasi-tests/src/bin/clock_time_get.rs

```diff
@@ -1,33 +1,16 @@
 use more_asserts::assert_le;
-use wasi::wasi_unstable;
-use wasi_tests::wasi_wrappers::wasi_clock_time_get;
 
 unsafe fn test_clock_time_get() {
     // Test that clock_time_get succeeds. Even in environments where it's not
     // desirable to expose high-precision timers, it should still succeed.
     // clock_res_get is where information about precision can be provided.
-    let mut time: wasi_unstable::Timestamp = 0;
-    let status = wasi_clock_time_get(wasi_unstable::CLOCK_MONOTONIC, 1, &mut time);
-    assert_eq!(
-        status,
-        wasi_unstable::raw::__WASI_ESUCCESS,
-        "clock_time_get with a precision of 1"
-    );
+    wasi::clock_time_get(wasi::CLOCKID_MONOTONIC, 1)
+        .expect("precision 1 should work");
 
-    let status = wasi_clock_time_get(wasi_unstable::CLOCK_MONOTONIC, 0, &mut time);
-    assert_eq!(
-        status,
-        wasi_unstable::raw::__WASI_ESUCCESS,
-        "clock_time_get with a precision of 0"
-    );
-    let first_time = time;
+    let first_time = wasi::clock_time_get(wasi::CLOCKID_MONOTONIC, 0)
+        .expect("precision 0 should work");
 
-    let status = wasi_clock_time_get(wasi_unstable::CLOCK_MONOTONIC, 0, &mut time);
-    assert_eq!(
-        status,
-        wasi_unstable::raw::__WASI_ESUCCESS,
-        "clock_time_get with a precision of 0"
-    );
+    let time = wasi::clock_time_get(wasi::CLOCKID_MONOTONIC, 0).unwrap();
```

Comment:
Is the `unwrap` here intentional?

Reply:
It is yeah, but only in the sense that it's an assertion about a successful syscall. Do you point this out in the sense that this should be an `expect`, not an `unwrap`?

Reply:
Yep, precisely, I expected an `expect` (pun not intended) here given that you've used `expect` everywhere else :-) I reckon `expect` would signal that we are intentionally unwrapping the value here.